config/initializers/active_fedora-noid.rb

```diff
@@ -0,0 +1,9 @@
+require 'active_fedora/noid'
+
+ActiveFedora::Noid.configure do |config|
+  config.statefile = '/tmp/minter-state'
```

Comment:
Should we change this default to `/var/sufia/minter-state` given the proposed default in InstallScripts?